lib/spack/spack/environment/environment.py

```diff
@@ -62,7 +62,7 @@
 _active_environment = None
 
 
-#: path where environments are stored in the spack tree
+#: default path where environments are stored in the spack tree
```

Comment:
I wonder if the variable name needs to be changed, perhaps `default_env_path`, given it would no longer be the only option?

Reply:
sure that makes sense to me